README.md

```diff
@@ -1,16 +1,311 @@
+  Claim Allocations:
+    1c8364bc-b45e-4b26-b74b-d24aea518319:
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-ee3e4b55-fcda-44b8-0605-64b7a9967744
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-9ede7e32-5825-a11b-fa3d-bab6d47e0243
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-e7b42cb1-4fd8-91b2-bc77-352a0c1f5747
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-657bd2e7-f5c2-a7f2-fbaa-0d1cdc32f81b
+    30a54813-002f-45f0-bac3-06ed401d31ad:
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-0159f35e-99ee-b2b5-74f1-9d18df3f22ac
+    37871101-7230-4b83-bda7-3d9e581ba515:
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-f11773a1-5bfb-e48b-3d98-1beb5baaf08e
+    7c6ca074-26ca-40b7-b71a-e06903ca13b7:
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-93d37703-997c-c46f-a531-755e3e0dc2ac
+    e16500f2-21c8-45e0-a6dc-280acfbcab4f:
+      Gpu:
+        Productname:  LATEST-GPU-MODEL
+        Uuid:         GPU-18db0e85-99e9-c746-8531-ffeb86328b39
+```
+
+Once you have verified everything is running correctly, delete all of the
+example apps:
+```bash
+kubectl delete --wait=false --filename=gpu-test{1,2,3,4}.yaml
+```
+
+Wait for them to terminate:
+```console
+$ kubectl get pod -A
+NAMESPACE   NAME   READY   STATUS        RESTARTS   AGE
+...
+gpu-test1   pod1   1/1     Terminating   0          31m
+gpu-test1   pod2   1/1     Terminating   0          31m
+gpu-test2   pod    2/2     Terminating   0          31m 
+gpu-test3   pod1   1/1     Terminating   0          31m 
+gpu-test3   pod2   1/1     Terminating   0          31m 
+gpu-test4   pod    1/1     Terminating   0          31m 
+...
+```
+
+And show that the `ClaimAllocations` section of the `NodeAllocationState`
+object on the worker node is now back to its initial state:
+```console
+$ kubectl describe -n dra-example-driver nas/dra-example-driver-cluster-worker
+...
+Spec:
+  Allocatable Devices:
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-657bd2e7-f5c2-a7f2-fbaa-0d1cdc32f81b
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-18db0e85-99e9-c746-8531-ffeb86328b39
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-93d37703-997c-c46f-a531-755e3e0dc2ac
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-ee3e4b55-fcda-44b8-0605-64b7a9967744
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-9ede7e32-5825-a11b-fa3d-bab6d47e0243
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-e7b42cb1-4fd8-91b2-bc77-352a0c1f5747
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-f11773a1-5bfb-e48b-3d98-1beb5baaf08e
+    Gpu:
+      Product Name:  LATEST-GPU-MODEL
+      Uuid:          GPU-0159f35e-99ee-b2b5-74f1-9d18df3f22ac
+...
+```
+
```

Comment:
May be it worth suggesting to run `./delete-cluster.sh` to clean it up.

Reply:
Good idea. Adding now.

Reply:
Updated